btcwallet.go

```diff
@@ -143,8 +143,39 @@ func walletMain() error {
 // associated with the server for RPC passthrough and to enable additional
 // methods.
 func rpcClientConnectLoop(legacyRPCServer *legacyrpc.Server, loader *wallet.Loader) {
-	var certs []byte
-	if !cfg.UseSPV {
+	var (
+		certs        []byte
+		chainService *neutrino.ChainService
+	)
+
+	if cfg.UseSPV {
```

Comment:
So this new change doesn't affect the `btcwallet` library at all, which is what we use ln `lnd` and elsewhere. If we go this route, seems that we'll need an additional set of modifications in `lnd` to use this new startup logic.